notebooks/building_production_ml_systems/solutions/1_training_at_scale_vertex.ipynb

```diff
@@ -61,7 +61,7 @@
     "%env BUCKET=$BUCKET\n",
```

Comment:
<p>This cell outputs an error because the the 'taxi' in the data path needs to be replaced by 'taxi-traffic'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
<p>+1</p>